webapp/webapp/webapp/static/admin/js/domains.js

```diff
@@ -411,7 +476,7 @@ $(document).ready(function() {
             $("#modalAddFromMedia #modalAdd")[0].reset();
             $("#modalAddFromMedia").modal('hide');   
             $("#modalTemplateDesktop #modalTemplateDesktopForm")[0].reset();
-            $("#modalTemplateDesktop").modal('hide');             
+            $("#modalTemplateDesktop").modal('hide');                       
```

Comment:
Could you remove the tailing whitespaces or not touch this line?
```suggestion
            $("#modalTemplateDesktop").modal('hide');
```